### PR TITLE
Fix few TCs and ops

### PR DESCRIPTION
**Pull request by nik-redhat**
Fix:
Fixed small list indices error and return value check  error in the TCs.

Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
Test run:
```
Table:
 test_peer_status
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|   Generic   |     PASS    | 30.459044933319092 |
+-------------+-------------+--------------------+


Framework runtime : 40.60669922828674
```
```
Table:
 test_shared_storage
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|   Generic   |     PASS    | 100.54803228378296 |
+-------------+-------------+--------------------+


Framework runtime : 112.68552374839783
```

This TC will fail in CI, because we are creating volume using `force` there.
```
Table:
 test_peer_probe
 
+-------------+-------------+-------------------+
| Volume Type | Test Result |  Time taken (sec) |
+-------------+-------------+-------------------+
|   Generic   |     PASS    | 61.47437238693237 |
+-------------+-------------+-------------------+


Framework runtime : 73.81182813644409
```

The rest of the TCs require 4 servers at least, hence not added the result.

**Review by nishith-vihar — APPROVED** [review]
LGTM

File: tests/functional/glusterd/test_volume_create.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def run_test(self, redant):
         ret = redant.setup_volume(self.volume_name1, self.server_list[0],
                                   conf_dict, self.server_list,
                                   self.brick_roots, excep=False)
-        if ret['error_code'] == 0:
+        if not ret:
```
Comment (aujjwal-redhat) [inline tests/functional/glusterd/test_volume_create.py:90]: Just had a doubt. ret is the dictionary right?

Comment (nik-redhat) [reply tests/functional/glusterd/test_volume_create.py:90]: This is a corner case, and we are expecting the `gluster vol list` output to show the volume list contains the volume already, and that will return a bool. Hence, updated the check accordingly. If this TC fails at this point due to type error, then anyway there is something going wrong in some other part, so we are good here.

Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_volume_create.py:90]: Oh ok. :+1:
